_projects/2019/astropy/telescopy.md

```diff
@@ -0,0 +1,50 @@
+---
+name: telescopy
```

Comment:
somehow this broke the listing on the website...

Reply:
@dpshelio How was it broken? I think I'm following the template correctly there.

Reply:
Yeah, I'm trying to figure that out too... the javascript complaining there, so that's why I think it's here. Not sure yet though...

Reply:
I think I've found the guilty one... it was not you!! It was the PR I merged just before your push to `master`. This means we have to improve our CI.

Reply:
Ok, this is now fixed on #198

Reply:
done!